sdk/iptables/iptables.go

```diff
@@ -0,0 +1,158 @@
+package iptables
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	"strconv"
+)
+
+const (
+	// Chain to intercept inbound traffic
+	ProxyInboundChain = "PROXY_INBOUND"
```

Comment:
What if we added a `CONSUL_` prefix to make it clear to users where these rules came from? Mostly for the VMs case where there may be other processes modifying `iptables` rules.

Reply:
Yeah, I like that a lot!